app/views/layouts/blacklight/base.html.erb

```diff
@@ -15,8 +15,11 @@
     <%= stylesheet_link_tag "https://ga.jspm.io/npm:dialog-polyfill@0.5.6/dist/dialog-polyfill.css" %>
     <% if defined? Importmap %>
       <%= javascript_importmap_tags %>
-    <% else %>
-      <%= javascript_include_tag "application", "data-turbo-track": "reload" %>
+    <% elsif defined? Propshaft %>
+      <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+    <% else %>    
+      <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
why `defer` here?

Reply:
Because we need this script to load before the `DOMContentLoaded` event fires.